Documentation/guides/port_arm_cm4.rst

```diff
@@ -0,0 +1,935 @@
+The case of ARM CM4 & cxd32xx @NuttX12.4.0
+==========================================
+
+Step1 : Add and register the SoC directory
+------------------------------------------
+
+Create the SoC directory.
+
+.. code-block::
+
+  $ touch arch/arm/include/cxd32xx
+  $ touch arch/arm/src/cxd32xx
+
+Modify "arch/arm/Kconfig" to register the SoC directory.
+
+Add "Kconfig" and "Make.defs" for the SoC directory.
+
+.. code-block::
+
+  $ touch arch/arm/src/cxd32xx/Kconfig
+  $ touch arch/arm/src/cxd32xx/Make.defs
+
+For details about these files, see the PR : https://github.com/apache/nuttx/pull/12441
```

Comment:
This is not nice. We should have good instructions instead of checking pull requests